src/playback/internal/playbackcontroller.cpp

```diff
@@ -310,13 +310,13 @@ void PlaybackController::setTrackSoloMuteState(const InstrumentTrackId& trackId,
     m_notation->soloMuteState()->setTrackSoloMuteState(trackId, state);
 }
 
-void PlaybackController::playElements(const std::vector<const notation::EngravingItem*>& elements)
+void PlaybackController::playElements(const std::vector<const notation::EngravingItem*>& elements, bool isMidi)
```

Comment:
maybe call the parameter "isEditing" to match the "playNotesWhenEditing" function, and to allow for still getting playback when using on-screen piano keyboard even if playNotesWhenEditing is off?